#467 : TransitionsService

SmolenskiyK opened the pull request.

Commit:
Transition service #467

msvaillant commented on src/wwwroot/app/views/candidate/candidate.controller.js:
```diff
@@ -240,17 +241,14 @@ export default function CandidateController( // eslint-disable-line max-params,
-      saveCandidate().then(() => {
-         $state.go('vacancyView', { vacancyId: vm.vacancyIdToGoBack, 'candidatesIds': [ vm.candidate.id ]});
+      saveCandidate().then((candidate) => {
+         TransitionsService.back(CANDIDATE_PROFILE_VIEW_NAME, { candidateId : candidate.id });
```
these changes are not equal!

msvaillant commented on src/wwwroot/app/views/vacancy/vacancy.edit.controller.js:
```diff
@@ -172,16 +173,14 @@ export default function VacancyController( //eslint-disable-line max-statements
+            vm.comments = cloneDeep(vm.vacancy.comments);
+            UserDialogService.notification($translate.instant('DIALOG_SERVICE.SUCCESSFUL_SAVING'), 'success');
+            SearchService.invalidateVacancies();
+            TransitionsService.back(DEFAULT_REDIRECT_VIEW_NAME, {vacancyId : vacancy.id});
```
think if this nice change, because in the previous case it was going in separate then

SmolenskiyK replied:
there are sync operations

Commit:
#467 TransitionsService fixes

in this commit fixes for TransitionsService and using this service in
all places where need
